bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -1,6 +1,6 @@
 def bubbleSort(arr): #Function that performs the bubbleSort
 
-  n = len(arr) #Finding the length of the array using the len function 
+  n = len(arr) #Finding the length of the array using the len function
 
   for i in range(n): # Traverse through all array elements 
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
```